MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -41,38 +41,84 @@
 import UIKit
 import FloatingPanel
 
+enum ScrollDirection {
+    case up
+    case down
+}
+
 protocol TransactionSelectedDelegate {
```

Comment:
I think this should be called `TransactionsTableViewDelegate` and moved to `TransactionsTableViewController.swift`.